just_audio/lib/just_audio.dart

```diff
@@ -2291,12 +2376,21 @@ class DashAudioSource extends UriAudioSource {
 /// your device to forward HTTP requests with headers included.
 class HlsAudioSource extends UriAudioSource {
   HlsAudioSource(Uri uri,
-      {Map<String, String>? headers, dynamic tag, Duration? duration})
-      : super(uri, headers: headers, tag: tag, duration: duration);
+      {Map<String, String>? headers,
+      List<AudioEffect>? effects,
+      dynamic tag,
+      Duration? duration})
```

Comment:
```suggestion
      Duration? duration,
      })
```
Controlla le virgole sui costruttori e sui metodi _toMessage